actionpack/CHANGELOG.md

```diff
@@ -1,11 +1,17 @@
-*   Fixes multiple calls to `logger.fatal` instead of a single call, 
-    for every line in an exception backtrace, when printing trace 
+*   Fixes Incorrect output from rails routes when using singular resources.
```

Comment:
Actually it changes the order of the output. I made a mistake writing "fix whitespaces" in a commit, i wanted to say that I erased some blank lines of a previous commit.